src/main/java/com/twitter/whiskey/net/SpdySession.java

```diff
@@ -670,7 +670,7 @@ private void sendData(SpdyStream stream) {
     private void sendClientSettings() {
 
         SpdySettings settings = new SpdySettings();
-        settings.setValue(SpdySettings.MAX_CONCURRENT_STREAMS, 100);
+        settings.setValue(SpdySettings.MAX_CONCURRENT_STREAMS, localMaxConcurrentStreams);
```

Comment:
Does the client work for non-push scenarios with the default of 0 in Builder.maxPushStreams?

src/main/java/com/twitter/whiskey/net/SpdySession.java:        localMaxConcurrentStreams = configuration.getMaxPushStreams();